imap_processing/cdf/config/imap_idex_l1a_variable_attrs.yaml

```diff
@@ -126,300 +136,616 @@ target_high_attrs:
         Data is used to quantify dust charge.
 
 ion_grid_attrs:
-  <<: *l1_target_base
+  <<: *l1a_target_base
   CATDESC: Ion grid charge sensitive amplifier waveform data
   FIELDNAM: Ion Grid Signal
   LABLAXIS: Ion Grid Ampl.
   VAR_NOTES: This is the ion grid signal from IDEX.
         Sampled at 3.75 Msps with 12-bit resolution.
         Data is used to quantify dust charge.
 
-# Trigger Stuff
-
-event_number:
+shcoarse:
   <<: *trigger_base
-  CATDESC: The unique number assigned to the impact by the FPGA
-  FIELDNAM: Event Number
-  VALIDMAX: 65535
-  LABLAXIS: Event Number
-  UNITS: " "
+  CATDESC: Time of packet generation (not data acquisition), as integer seconds since epoch
+  FIELDNAM: Secondary header coarse time
+  LABLAXIS: Packet Generation Time (Coarse)
+  UNITS: seconds
 
-tof_high_trigger_level:
+shfine:
   <<: *trigger_base
-  VALIDMAX: 1023
-  FIELDNAM: TOF High Trigger Level
-  CATDESC: Trigger level for the TOF High Channel
-  LABLAXIS: Level
-  UNITS: " "
+  CATDESC: Time of packet generation, each DN represents 20usec within current second
+  FIELDNAM: Secondary header fine time
+  VALIDMAX: *max_uint16
+  LABLAXIS: Packet Generation Time (Fine)
+  UNITS: dN
```

Comment:
This should be seconds or subseconds?

Reply:
Yes I think you're right. Ill do seconds